### PR TITLE
Update and rename README to README.md

### PR DESCRIPTION
**Pull request by Oxicode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,15 @@
-usage : tty-clock [-iuvsScbtrahDBxn] [-C [0-7]] [-f format] [-d delay] [-a nsdelay] [-T tty] 
+## Instalation
+```
+cd /tmp
+wget https://github.com/xorg62/tty-clock/archive/master.zip
+unzip master.zip
+cd tty-clock-master/
+make
+sudo mv tty-clock /usr/bin/tty-clock
```
Comment (anarcat) [inline README.md]: this should be make install

File: README.md [hunk 2]
```diff
@@ -1,4 +1,15 @@
-usage : tty-clock [-iuvsScbtrahDBxn] [-C [0-7]] [-f format] [-d delay] [-a nsdelay] [-T tty] 
+## Instalation
+```
+cd /tmp
+wget https://github.com/xorg62/tty-clock/archive/master.zip
```
Comment (anarcat) [inline README.md]: there are releases now - not sure how to get the latest version? maybe tell people to use git instead...

File: README.md [hunk 3]
```diff
@@ -1,4 +1,15 @@
...
+unzip master.zip
+cd tty-clock-master/
+make
+sudo mv tty-clock /usr/bin/tty-clock
+```
+
+##Usage : 
+tty-clock [-iuvsScbtrahDBxn] [-C [0-7]] [-f format] [-d delay] [-a nsdelay] [-T tty]
```
Comment (livibetter) [inline README.md]: This section needs to be in literal block for better formatting.

File: README.md (hunk 1 above)
Comment (livibetter) [inline README.md]: and also mention `uninstall` target.

**Comment by anarcat** [comment]
ping? open to making changes?

**Pull request closed without merging**